doc/tutorial/introduction/introduction_read_fasta.cpp

```diff
@@ -32,8 +32,6 @@ int main(int argc, char * argv[])
     for (auto & [ seq, id, qual ] : file_in)
```

Comment:
```suggestion
    for ([[maybe_unused]] auto & [ seq, id, qual ] : file_in)
```
Does this work?

Reply:
No. There is no way to use [[maybe_unused]] with structured bindings in C++17.

Reply:
At least not for gcc-7: https://stackoverflow.com/questions/41404001/structured-binding-with-maybe-unused I am afraid. 
So in this case don't do structured binding.

Reply:
sorry, I didn't see this reply when merging.